### PR TITLE
Before you begin hyperlink and copy update #962

### PR DESCRIPTION
**Pull request by awellsbiz**
Hi,

For issue #962, I have updated the hyperlinks for mobile and made the design changes per the figma document provided [here.](https://www.figma.com/file/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?type=design&node-id=10739%3A112003&t=JFzXfKekw4yZXp0T-1)

### CODE REVIEW

**Review by sydneywalcoff — CHANGES_REQUESTED** [review]
Hi @awellsbiz!

Thanks for submitting a PR! 🎉 I appreciate the quick turn-around.

Just a couple code corrections above. Otherwise, I noticed that some of the font-sizes are off. Can you double-check them?

Thank you! Slack me if you have any questions!

File: products/statement-generator/public/locales/en/translation.json [hunk 1]
```diff
@@ -45,8 +45,9 @@
...
-    "sectionTitle1": "Eligibility",
-    "sectionParagraph1": "Be sure you are eligible to have your record cleared. If you’re not sure, here are <a target='_blank' href='https://roadmap.rootandrebound.org/understanding-cleaning-up-your-criminal-record/cleaning-up-your-criminal-record-later-in-reentry/california-expungement-of-state-convictions/chart-which-type-of-expungement-am-i-eligible-for/'>two</a> <a target='_blank' href='https://www.robertmhelfend.com/criminal-defense/how-get-felony-expunged-california/#6-step-2-find-out-if-you-are-eligible-for-expungement'>resources</a> to help.",
+    "sectionTitle1": "Are you eligible?",
+    "sectionParagraph1": "Be sure you are eligible to have your record cleared. If you’re not sure, the <a target='_blank' href='https://selfhelp.courts.ca.gov/clean-your-record'>California Courts Self-Help Guide</a> or the <a target='_blank' href='https://roadmap.rootandrebound.org/understanding-cleaning-up-your-criminal-record/cleaning-up-your-criminal-record-later-in-reentry/california-expungement-of-state-convictions/chart-which-type-of-expungement-am-i-eligible-for/'>Roadmap to Reentry</a> can help you get started.",
```
[the remark this answers is not in the document]
Comment (awellsbiz) [reply products/statement-generator/public/locales/en/translation.json:49]: copy will work on this today

**Comment by awellsbiz** [comment]
Hi @sydneywalcoff 

The changes have been made. Thanks for the CSS tips 😃

**Review by sydneywalcoff — APPROVED** [review]
@awellsbiz Thanks for getting these changes in! I'll get this merged

**Pull request merged**